### ISSUE TITLE
Make a library to have the generateQuery function in the window object on the browser side

### ISSUE THREAD

**Issue by MurzNN** [john-yuan/graphql-toolkit#1]
Generating GraphQL queries is pretty often needed not only in Node.js projects with transpilers like Babel but also in regular scripts that run in the browser.

But the currently released version of the script https://unpkg.com/generate-graphql-query@1.0.0/es/index.js is more for the backend transpilers, than for the browsers, because the function is available only in the export context.

So, to properly get this function in custom scripts in the browser context, we should use a trick like this:
```html
<html>
<head>
    <script type="module">
        import generateQuery from 'https://unpkg.com/generate-graphql-query@1.0.0/es/index.js';
        window.graphqlToolkit ??= {};
        window.graphqlToolkit.generateQuery = generateQuery;
    </script>
    <script src="/myScript.js"></script>
</head>
<body>
</body>
</html>
```
and the `myScript.js`:
```js
const query = graphqlToolkit.generateQuery({
  query: {
    countries: {
      code: true,
      name true,
    }
  }
})

console.log(query)
```

And would be good to simplify this by just loading the script directly, something like this:
```html
<html>
<head>
    <script src="https://unpkg.com/generate-graphql-query@1.0.0/browser/index.js"></script>
    <script src="/myScript.js"></script>
</head>
<body>
</body>
</html>
```

If this already can be done using the current release, could you please provide an example?

**Comment by john-yuan** [comment]
Thank you for your suggestion. We've just released version `1.1.0`, which allows us to load the script directly from a CDN like unpkg. For example:

```html
<script src="https://unpkg.com/generate-graphql-query/browser/index.js"></script>
<script>
  var query = GraphQLToolkit.generateQuery({
    query: {
      countries: {
        code: true,
        name: true
      }
    }
  })

  console.log(query)
</script>
```

Please note that the global variable name is `GraphQLToolkit`.

**Comment by MurzNN** [comment]
Thank you a lot for such a quick fix! Tested - it works well!
